tests/test_e2e_10_mef_eline.py

```diff
@@ -1369,3 +1369,121 @@ def test_155_removing_evc_metadata_persistent(self):
         data = response.json()
         assert my_key not in data['metadata']
         assert len(data['metadata']) > 0
+
+    def test_160_create_untagged_evc(self):
+        payload = {
+            "name": "evc1",
+            "dynamic_backup_path": True,
+            "enabled": True,
+            "uni_a": {
+                "tag": {"tag_type": 1, "value": "untagged"},
+                "interface_id": "00:00:00:00:00:00:00:01:1"
+            },
+            "uni_z": {
+                "tag": {"tag_type": 1, "value": "untagged"},
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            }
+        }
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        assert response.status_code == 201, response.text
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(20)
+
+        h11, h2 = self.net.net.get('h11', 'h2')
+        h11.cmd('ip addr add 104.0.0.11/24 dev %s' % (h11.intfNames()[0]))
+        h2.cmd('ip addr add 104.0.0.2/24 dev %s' % (h2.intfNames()[0]))
+        result = h11.cmd('ping -c1 104.0.0.2')
+        assert ', 0% packet loss,' in result
+        
+    def test_165_create_any_evc(self):
+        payload = {
+            "name": "evc1",
+            "dynamic_backup_path": True,
+            "enabled": True,
+            "uni_a": {
+                "tag": {"tag_type": 1, "value": "any"},
+                "interface_id": "00:00:00:00:00:00:00:01:1"
+            },
+            "uni_z": {
+                "tag": {"tag_type": 1, "value": "any"},
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            }
+        }
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        assert response.status_code == 201, response.text
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(10)
+
+        h11, h2 = self.net.net.get('h11', 'h2')
+        h11.cmd('ip link add link %s name vlan100 type vlan id 100' % (h11.intfNames()[0]))
```

Comment:
Since we are using the "any" VLAN here, what do you think about using a random value for the VLAN on the PING test? I suggest replacing the VLAN 100 with a random value chosen from a range from 1 to 4095. Let me know what you think.

Reply:
Sure, will do that.

Reply:
I divided the PR, here is the correct one for [mef_eline](https://github.com/kytos-ng/kytos-end-to-end-tests/pull/213)